core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -155,29 +161,31 @@ public ClusterConfig copyOf() {
     }
 
     public static Builder defaultClusterBuilder() {
-        return new Builder(Type.ZK, 1, 1, true, SecurityProtocol.PLAINTEXT, MetadataVersion.latestTesting());
+        return new Builder(Type.ZK, 1, 1, 1, true, SecurityProtocol.PLAINTEXT, MetadataVersion.latestTesting());
     }
 
-    public static Builder clusterBuilder(Type type, int brokers, int controllers, boolean autoStart,
+    public static Builder clusterBuilder(Type type, int brokers, int controllers, int disksPerBroker, boolean autoStart,
                                          SecurityProtocol securityProtocol, MetadataVersion metadataVersion) {
-        return new Builder(type, brokers, controllers, autoStart, securityProtocol, metadataVersion);
+        return new Builder(type, brokers, controllers, disksPerBroker, autoStart, securityProtocol, metadataVersion);
     }
 
     public static class Builder {
         private Type type;
         private int brokers;
         private int controllers;
+        private int disksPerBroker;
         private String name;
         private boolean autoStart;
         private SecurityProtocol securityProtocol;
         private String listenerName;
         private File trustStoreFile;
         private MetadataVersion metadataVersion;
 
-        Builder(Type type, int brokers, int controllers, boolean autoStart, SecurityProtocol securityProtocol, MetadataVersion metadataVersion) {
+        Builder(Type type, int brokers, int controllers, int disksPerBroker, boolean autoStart, SecurityProtocol securityProtocol, MetadataVersion metadataVersion) {
```

Comment:
Could we call setters instead of having a complicated constructor? We should not waste the build pattern ... this is unrelated to this PR, and it already have a jira (https://issues.apache.org/jira/browse/KAFKA-16560). However, we can do a bit refactor here.